Show upscaled resolution on hires fix

Mitchell1711 opened the pull request:
**Issue**
The new hires fix doesn't show what the resolution will be of your final image, this is a bit inconvenient as you will need to manually calculate what the resolution will be after upscaling.

**What I added**
I added a new row to the hires fix which shows what the resolution of the image will be after upscaling, this updates whenever the "Upscale by", "Width" or "Height" sliders are changed.
![image](https://user-images.githubusercontent.com/47387831/211121069-0abc66e7-b4d0-47ba-aecc-e147dd3e9a0a.png)

Mitchell1711 committed:
Added UI elements

Added a new row to hires fix that shows the new resolution after scaling

Mitchell1711 committed:
remove camelcase

ice051128 commented:
Or just bring back the old hi-res fix since the new one is just a waste of time when u have to do these pointless calculations. Just specify the target resolution for the final output and bam done

freecoderwaifu commented:
This is a good change, really the only final thing the new highres fix needs to go from 99.9% perfect to 100%, especially for people like me who can't math.

> Or just bring back the old hi-res fix since the new one is just a waste of time when u have to do these pointless calculations. Just specify the target resolution for the final output and bam done

You already needed to do calculations for non square ratios if you didn't want your images cropped, you might just have never noticed it was cropping them. The new method is much better.

Mitchell1711 committed:
Changed HTML to textbox instead

Using HTML caused an issue where the row would expand for a frame when changing the sliders because of the loading animation. This solution also doesn't use any additional HTML padding

AUTOMATIC1111 commented:
That was overall a very raw feature by you mitchell 1711. It make webui not launch on default config because width and height were None at the point where you used them as input. In my rework I also placed label to the same line where checkboxes are to take less space, and made made the calculation use existing functions - the correct calculation for resulting dimensions has to take five sliders into account: width, height, upscale by, resize width to, resize height to.

![firefox_xA0IjqCEOD](https://user-images.githubusercontent.com/20920490/211135609-6127bb7a-fa9a-4442-a90b-2a10ec406a6a.png)
